corehq/apps/hqwebapp/views.py

```diff
@@ -83,6 +86,11 @@ def couch_check():
         return isinstance(results, list), None
 
 
+def is_deploy_in_progress():
+    cache = get_redis_default_cache()
```

Comment:
I think we'd also want to be explicit about this and only set / get this value from the redis cache (not any other default cache that might be defined), unless we can guarantee the same set and expire functionality when setting the flag for each cache that might be used.